### ISSUE TITLE
How to force overlay true?

### ISSUE THREAD

**Issue by abdonrd** [vaadin/vaadin-app-layout#158]
I want to force the `overlay` option to be always `true` instead of relying on the media query.

Is there any option to do it?

**Comment by web-padawan** [comment]
Are you using latest stable version or LitElement based alpha?

In the Polymer version, you can force it by setting custom CSS property directly:

```js
layout.style.setProperty('--vaadin-app-layout-drawer-overlay', 'true');
```

In the LitElement version, we changed that so media query has to be used. For now it is hardcoded, we should expose it as a configurable CSS custom property in future.

**Comment by abdonrd** [comment]
Sorry! Yes, I'm using the `v3.0.0-alpha2`.

**Comment by web-padawan** [comment]
Technically it's still possible to set `overlay` as an attribute, if that is needed for styling.
This is actually how the [demo](https://cdn.vaadin.com/vaadin-app-layout/3.0.0-alpha2/index.html) works (check `overlay` in Knobs to see it in action).

**Comment by abdonrd** [comment]
> Technically it's still possible to set `overlay` as an attribute, if that is needed for styling.
> This is actually how the [demo](https://cdn.vaadin.com/vaadin-app-layout/3.0.0-alpha2/index.html) works (check `overlay` in Knobs to see it in action).

But if the viewport changes, the mediaProperty will update the value, right?

**Comment by web-padawan** [comment]
> But if the viewport changes, the mediaProperty will update the value, right?

Yes, that's how it works for now. You can use a really dirty workaround to disable that, by tweaking [internal property](https://github.com/vaadin/component-mixins/blob/c83bb8903ef1a84e011035b3ba30a99dfbc98e7f/packages/media-query-mixin/media-query-mixin.ts#L49) and then setting [](https://github.com/vaadin/component-mixins/blob/c83bb8903ef1a84e011035b3ba30a99dfbc98e7f/packages/media-query-mixin/media-query-mixin.ts#L55) and then [removing listener](https://github.com/vaadin/component-mixins/blob/c83bb8903ef1a84e011035b3ba30a99dfbc98e7f/packages/media-query-mixin/media-query-mixin.ts#L98) manually 🙈
